tests/jballerina-unit-test/src/test/resources/test-src/bir/bir-dump/mapInits

```diff
@@ -50,211 +50,212 @@ mapInits function() -> (string|(), int|()) {
     }
     bb24 {
         %12 = ConstLoad jack;
-        %73 = %1[%12];
-        %77 = %73;
-        %29 = ConstLoad true;
-        %29? bb25 : bb26;
+        %74 = %1[%12];
+        %78 = %74;
+        %30 = ConstLoad true;
+        %30? bb25 : bb26;
     }
     bb25 {
-        %80 = ConstLoad true;
-        %81 = <any|error> %77;
+        %81 = ConstLoad true;
+        %82 = <any|error> %78;
         GOTO bb27;
     }
     bb26 {
-        %80 = ConstLoad false;
+        %81 = ConstLoad false;
         GOTO bb27;
     }
     bb27 {
-        %80? bb28 : bb29;
+        %81? bb28 : bb29;
     }
     bb28 {
-        %79 = %81 is ();
+        %80 = %82 is ();
         GOTO bb30;
     }
     bb29 {
-        %79 = ConstLoad false;
+        %80 = ConstLoad false;
         GOTO bb30;
     }
     bb30 {
-        %79? bb31 : bb32;
+        %80? bb31 : bb32;
     }
     bb31 {
-        %72 = <int|()> %81;
+        %73 = <int|()> %82;
         GOTO bb48;
     }
     bb32 {
-        %29 = ConstLoad true;
-        %29? bb33 : bb34;
+        %30 = ConstLoad true;
+        %30? bb33 : bb34;
     }
     bb33 {
-        %95 = ConstLoad true;
-        %96 = <any|error> %77;
+        %96 = ConstLoad true;
+        %97 = <any|error> %78;
         GOTO bb35;
     }
     bb34 {
-        %95 = ConstLoad false;
+        %96 = ConstLoad false;
         GOTO bb35;
     }
     bb35 {
-        %95? bb36 : bb37;
+        %96? bb36 : bb37;
     }
     bb36 {
-        %94 = %96 is Employee;
+        %95 = %97 is Employee;
         GOTO bb38;
     }
     bb37 {
-        %94 = ConstLoad false;
+        %95 = ConstLoad false;
         GOTO bb38;
     }
     bb38 {
-        %94? bb39 : bb40;
+        %95? bb39 : bb40;
     }
     bb39 {
-        %13 = <Employee> %96;
+        %13 = <Employee> %97;
         %7 = ConstLoad age;
-        %108 = %13[%7];
-        %72 = <int|()> %108;
+        %109 = %13[%7];
+        %73 = <int|()> %109;
         GOTO bb48;
     }
     bb40 {
-        %29 = ConstLoad true;
-        %29? bb41 : bb42;
+        %30 = ConstLoad true;
+        %30? bb41 : bb42;
     }
     bb41 {
-        %113 = ConstLoad true;
+        %114 = ConstLoad true;
         GOTO bb43;
     }
     bb42 {
-        %113 = %77 is any;
+        %114 = %78 is any;
         GOTO bb43;
     }
     bb43 {
-        %113? bb44 : bb45;
+        %114? bb44 : bb45;
     }
     bb44 {
-        %112 = ConstLoad true;
+        %113 = ConstLoad true;
         GOTO bb46;
     }
     bb45 {
-        %112 = ConstLoad false;
+        %113 = ConstLoad false;
         GOTO bb46;
     }
     bb46 {
-        %112? bb47 : bb48;
+        %113? bb47 : bb48;
     }
     bb47 {
-        %70 = ConstLoad 0;
-        %72 = <int|()> %70;
+        %71 = ConstLoad 0;
+        %73 = <int|()> %71;
         GOTO bb48;
     }
     bb48 {
-        %0 = newArray (string|(), int|())[%10];
+        %0 = newArray %2[%10];
         GOTO bb49;
     }
     bb49 {
         return;
     }
 
 
-}
\ No newline at end of file
+}
```

Comment:
missing last line

Reply:
The last line has been added with this PR.